user_docs/en/changes.t2t

```diff
@@ -18,6 +18,9 @@ What's New in NVDA
 == Bug Fixes ==
 - Reporting of object shortcut keys has been improved. (#10807)
 - The SAPI4 synthesizer now properly supports volume, rate and pitch changes embedded in speech. (#15271)
+- Braille is updated when control+v, control+x, control+z or backspace is pressed.

```

Comment:
Please modify this entry, so that it is clear these changes affect only MS Word.